### PR TITLE
feat: add giot.light.v8ssm

### PR DESCRIPTION
**Pull request by lekoOwO**

### CODE REVIEW

**Review by AlexxIT — APPROVED** [review]

**Comment by AlexxIT** [comment]
Thanks!

File: custom_components/xiaomi_gateway3/core/devices.py [hunk 1]
```diff
@@ -3071,6 +3071,7 @@
     "ttl": "7d",
 }, {
     11724: ["GranwinIoT", "Mesh Light V5", "giot.light.v5ssm"],
+    15504: ["GranwinIoT", "Mesh Light V8", "giot.light.v8ssm"]
```
Comment (Necroneco) [inline custom_components/xiaomi_gateway3/core/devices.py:3074]: forgot a comma

Comment (lekoOwO) [reply custom_components/xiaomi_gateway3/core/devices.py:3074]: ono

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/XiaomiGateway3/releases/tag/v4.0.6

**Pull request merged**